pkgdb/tests/search.bats

```diff
@@ -578,6 +578,7 @@ genParamsNixpkgsFlox() {
 
 # ---------------------------------------------------------------------------- #
 
+# bats test_tags=search:parallel
```

Comment:
You'd want to back this out too if you get rid of the second parallel test

Reply:
Test tags are useful anyway, i dont think its too misplaced tbh